portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,6 +45,34 @@ public void init() {
         countries.add(new Country("South Africa (any one of the 3 capitals)", "Bloemfontein, Cape Town, Pretoria"));
     }
 
+    public void jsonInit() {
+        hardCodedJson.add("This is my first comment!");
```

Comment:
Looks like `hardCodedJson` is really a list of `hardCodedComments` -- this was a little unclear.

I'm assuming you'll replace this with the DataStore in the next part?